assessment/scripts/3_hwe_analysis.py

```
@@ -0,0 +1,235 @@
+            f.write(f"{vcf_path}\n")
+
+    # Merge VCFs using bcftools
+    cmd = [
+        "bcftools", "merge",
+        "-l", str(file_list),
+        "-m", "none",  # Don't merge variant records
+        "-Oz",
+        "-o", str(output_vcf)
+    ]
+
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)